### ISSUE TITLE
Jisho search doesn't open with whole card content

### ISSUE THREAD

**Issue by devExcale** [AustinHasten/AnkiKanjiTable#3]
Hi, this is related to #2 (same causation).

I'm using a deck containing words with multiple kanji, clicking on a cell opens the jisho search for the first character only, disregarding the remaining characters (and thus making it impossible to understand the referenced card).

Would it be possible to make it so that the jisho search includes all characters? (but not meta-characters like HTML tags in #1)

Thanks

**Comment by AustinHasten** [comment]
Could you give an example? The newest version should now create a tile for every kanji in the word, so "国家" will create "国" and "家". Are you saying that then clicking on either "国" or "家" should open Jisho for "国家" instead of either of the individual kanji? This would be difficult, since if you had "作家" as well as "国家", then how do we decide what word to search for when you click on "家"?

**Comment by devExcale** [comment]
Yes, the new update solved the issue, thanks.

(Plugin version 2024-01-06)

**Comment by devExcale** [comment]
The issue solved itself with the new version (2024-01-06), where as you said multiple-kanji words are split in single kanji with no duplicates; cells now reference a single kanji (not cards), so there's no need anymore for the suggested feature.

If I understood correctly, in the previous version each cell referenced a single card but just the first character, so words that started with the same kanji (e.g. 四, 四季, 四つ角) would show three duplicate cells (e.g. 四, 四, 四) that all pointed to the jisho search for 四 only. But the new version splits the kanji and makes a cell for each kanji with no duplicates, so the issue is solved, thanks.

TL;DR new version fixed the issue

**Comment by AustinHasten** [comment]
I see what you mean now. Glad this is now working as you expect!
